code/src/App.js

```diff
@@ -1,12 +1,56 @@
 import React from 'react'
 import data from './data.json'
+import { Albums } from './Components/Albums'
+import { Heading } from './Components/Heading'
+import { Artists } from './Components/Artists'
+import { AlbumTitle } from './Components/AlbumTitle'
+import { TypeOf } from './Components/TypeOf'
+import { Footer } from './Components/Footer'
+import './App.css'
```

Comment:
There is a convention that says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, style.css should be separated as well and located below local dependencies.